cmd/osv-scanner/scan/main.go

```diff
@@ -221,6 +229,18 @@ func action(context *cli.Context, stdout, stderr io.Writer) (reporter.Reporter,
 		callAnalysisStates = createCallAnalysisStates(context.StringSlice("call-analysis"), context.StringSlice("no-call-analysis"))
 	}
 
+	offlineVulns := context.Bool("experimental-offline-vulnerabilities")
+	disableTransitive := context.Bool("experimental-no-resolve")
+	if context.Bool("experimental-offline") {
+		// Set the corresponding offline flags, unless they were already set explicitly.
+		if !context.IsSet("experimental-offline-vulnerabilities") {
+			offlineVulns = true
+		}
```

Comment:
The `CompareOffline` field that `experimental-offline-vulnerabilities` sets also affects the `SkipGit` and license scanning  in `osvscanner.DoScan`.
I think it might be a bit clearer to instead set them here similarly, but that could create a subtle change in behaviour in `DoScan`.